orca-clouddriver/src/main/java/com/netflix/spinnaker/orca/clouddriver/MortService.java

```diff
@@ -0,0 +1,234 @@
+/*
```

Comment:
This whole class is uh.. fun.

Unfortunately whereas in Groovy we have ?: to gracefully handle null checking for nested objects, such a thing would be too convenient for Java. Because pretty much everything in these services is just a `Map` rather than strongly typed it makes it basically impossible to do any proper checks.

I'd migrate these Maps to be strongly typed but I'm not familiar enough with what the payloads look like for all the different cloud providers that use them.

Reply:
Thanks for the heads-up here...and for wading through it.